### PR TITLE
Appw refine validation, improve message

### PR DESCRIPTION
**Pull request by brutif**

### CODE REVIEW

**Comment by brutif** [comment]
#build
#run-libby-bot
#spawn.fullfat.buckets=com.ibm.ws.security.openidconnect.server_fat.endpoint.app-passwords.DerbyDB,com.ibm.ws.security.openidconnect.server_fat.endpoint.app-passwords.DerbyDB.jwt,com.ibm.ws.security.openidconnect.server_fat.endpoint.app-passwords.DerbyDB.opaque,com.ibm.ws.security.openidconnect.server_fat.endpoint.app-passwords.LocalStore,com.ibm.ws.security.openidconnect.server_fat.endpoint.app-passwords.LocalStore.jwt,com.ibm.ws.security.openidconnect.server_fat.endpoint.app-passwords.LocalStore.opaque

File: dev/com.ibm.ws.security.oauth/resources/com/ibm/ws/security/oauth20/resources/ProviderMsgs.nlsprops [hunk 1]
```diff
@@ -322,7 +322,7 @@ security.oauth20.front.end.error=CWOAU0073E: An authentication error occurred. T
 security.oauth20.front.end.error.explanation=The request might be missing information, the server might have encountered an internal error, or some other error may have prevented the server from successfully authenticating the user.
 security.oauth20.front.end.error.useraction=Contact the system administrator to resolve the problem.
 
-security.oauth20.endpoint.resowner.apppassword.error=CWOAU0074E: The application password exchange request for the {0} user was not completed because the application password was not able to be verified. The application password is either incorrect, expired, or revoked, or the provided client ID is not valid.
+security.oauth20.endpoint.resowner.apppassword.error=CWOAU0074E: The application password exchange request for the {0} user was not completed because the application password was not able to be verified. The application password is either incorrect, expired, or revoked, or the provided client ID or user ID is not valid.
```
[the remark this answers is not in the document]
Comment (brutif) [reply dev/com.ibm.ws.security.oauth/resources/com/ibm/ws/security/oauth20/resources/ProviderMsgs.nlsprops]: done

**Comment by brutif** [comment]
#run-libby-bot
#build

**Comment by skasund** [comment]
L2 message review completed

**Pull request merged**